content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx

```diff
@@ -43,7 +41,7 @@ We recommend that you keep **both** the `prisma` and `@prisma/client` packages i
 1. You can now [instantiate the Prisma Client](instantiate-prisma-client) in your code:
 
    <TabbedContent tabs={[<FileWithIcon text="TypeScript" icon="code"/>, <FileWithIcon text="JavaScript" icon="code"/>]}>
-   <tab>
+<tab>
```

Comment:
Hm this looks wrong in this case.

Reply:
Same problem with markdown lists and JSX